RealmSwift/Util.swift

```diff
@@ -160,14 +160,16 @@ internal func logRuntimeIssue(_ message: StaticString) {
     }
 }
 
-#if swift(<5.9)
```

Comment:
The API ended up getting changed after I wrote this, but it turns out we can't unconditionally use the built-in versions anyway due to the deployment target.